app/commands/v2/unpublish.rb

```diff
@@ -132,7 +132,9 @@ def find_unpublishable_content_item(content_id)
         end
 
         filter = ContentItemFilter.new(scope: ContentItem.where(content_id: content_id))
-        filter.filter(locale: locale, state: allowed_states).first
```

Comment:
When I have looked in to other issues related to the unpublishing change, I did look at whether `.first` could be changed to something which would error if there is more than one result. I didn't find anything simple, but it may be worth adding an explicit check here?